### PR TITLE
Call setWasNull in GroupByStreamMergedResult

### PR DESCRIPTION
**Pull request by plazmdk**
Changes proposed in this pull request:
- Set wasNull field which had wrong value when doing a LEFT JOIN FETCH via Hibernate resulting in trying getting an Entity with ID 0.

 See demo application
[demoapp.zip](https://github.com/apache/incubator-shardingsphere/files/4159223/demoapp.zip)

### CODE REVIEW

File: sharding-core/sharding-core-merge/src/main/java/org/apache/shardingsphere/sharding/merge/dql/groupby/GroupByStreamMergedResult.java [hunk 1]
```diff
@@ -129,11 +129,15 @@ private void setAggregationValueToCurrentRow(final Map<AggregationProjection, Ag
     
     @Override
     public Object getValue(final int columnIndex, final Class<?> type) {
-        return currentRow.get(columnIndex - 1);
+        Object o = currentRow.get(columnIndex - 1);
```
Comment (tuohai666) [inline sharding-core/sharding-core-merge/src/main/java/org/apache/shardingsphere/sharding/merge/dql/groupby/GroupByStreamMergedResult.java]: o -> object

File: sharding-core/sharding-core-merge/src/main/java/org/apache/shardingsphere/sharding/merge/dql/groupby/GroupByStreamMergedResult.java [hunk 2]
```diff
@@ -129,11 +129,15 @@ private void setAggregationValueToCurrentRow(final Map<AggregationProjection, Ag
...
+        setWasNull(o == null);
```
Comment (tuohai666) [inline sharding-core/sharding-core-merge/src/main/java/org/apache/shardingsphere/sharding/merge/dql/groupby/GroupByStreamMergedResult.java]: o == null -> null == object

**Comment by tuohai666** [comment]
@plazmdk  You are right. GroupByStreamMergedResult overwrite the getValue() method but miss set the wasNull value.
Can you fix the code conventions above in ShardingSphere(https://shardingsphere.apache.org/community/en/contribute/code-conduct/)?
I thinks this pr is also applicated to the master branch. Can you submit another more pr to master?

**Comment by plazmdk** [comment]
I believe the code convensions are followed now - and another pr have been made for master

**Comment by tuohai666** [comment]
You are right, the return value should named result.

**Pull request merged**